private/api/2ptsToProfile.py

```diff
@@ -8,10 +8,10 @@
 # Setting axes to "xyz" will return image coordinates instead of lat lons
 
 import sys
-import gdal
-import osr
 import math
-from gdalconst import *
+from osgeo import gdal

```

Comment:
Can you verify that `from osgeo import gdal` works from the docker container.

Reply:
Accidentally set it as requested changes but it's just a comment and you can merge once verified.

Reply:
I built a Docker container with the latest changes and the scripts worked as expected.